src/NServiceBus.Msmq.AcceptanceTests/NServiceBusAcceptanceTest.cs

```diff
@@ -22,12 +24,18 @@ public void SetUp()
                 testName = testName.Replace("When_", "");
 
                 var endpointBuilder = classAndEndpoint.Split('+').Last();
-                
+
                 testName = System.Threading.Thread.CurrentThread.CurrentCulture.TextInfo.ToTitleCase(testName);
                 testName = testName.Replace("_", "");
 
                 return testName +"."+ endpointBuilder;
             };
         }
+
+        [TearDown]
+        public void TearDown()
```

Comment:
no, good point. I kept it there to make sure all instance mappings leftovers are removed on the buildserver